taglib/asf/asffile.h

```diff
@@ -134,6 +134,7 @@ namespace TagLib {
       class ContentDescriptionObject;
       class ExtendedContentDescriptionObject;
       class HeaderExtensionObject;
+      class CodecListObject;
```

Comment:
I realized it's not so complicated. I just moved the forward declarations to ```FilePrivate```, and it works fine.
Actually, we can do that in this pull request.